instrumentation/micronaut-1.0/build.gradle.kts

```diff
@@ -21,28 +20,32 @@ dependencies {
     testAnnotationProcessor("io.micronaut:micronaut-inject-java:${micronautVersion}")
 }
 
-/**
- * Todo this does not run tests
- */
-for (version in listOf(micronautVersion, micronaut2Version)) {
+val micronaut2Version = "2.2.3"
+
+for (version in listOf(micronaut2Version)) {
     val versionedConfiguration = configurations.create("test_${version}") {
         extendsFrom(configurations.testRuntimeClasspath.get())
     }
     dependencies {
         versionedConfiguration(project(":instrumentation:netty:netty-4.1"))
```

Comment:
These aren't present in the `testImplementation` either - I'd expect the dependencies called out here to match those, except with different versions (and the added junit ones which are added externally from your root file for `testImplementation` and aren't visible in this file)

Reply:
not needed removing